Jenkinsfile

```diff
@@ -294,6 +294,7 @@ void applyCustomizations() {
 void runTests() {
   sh '''
     export PIP_INDEX_URL=https://${ARTIFACTORY_CREDS_USR}:${ARTIFACTORY_CREDS_PSW}@${ARTIFACTORY_URL_DOWN##'https://'}/api/pypi/cloudant-sdks-pypi-virtual/simple
+    python3 -m tox -e py39-lint
```

Comment:
I'd like to see the lint env updated to python 3.11 so it works on our current test python and doesn't rely on a matching 3.9 system python (which could change on the build executors at any time) - hopefully that is a simple change in the tox.ini.

This Jenkinsfile alteration itself will need to happen in the gen branch.